x-pack/legacy/plugins/alerting/server/shim.ts

```diff
@@ -0,0 +1,143 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import Hapi from 'hapi';
+import { Legacy } from 'kibana';
+import { SpacesPlugin as SpacesPluginStartContract } from '../../spaces';
+import { ActionsPlugin } from '../../actions';
+import { TaskManager } from '../../task_manager';
+import { XPackMainPlugin } from '../../xpack_main/xpack_main';
+import KbnServer from '../../../../../src/legacy/server/kbn_server';
+import { EncryptedSavedObjectsPlugin } from '../../encrypted_saved_objects';
+import { PluginSetupContract as SecurityPlugin } from '../../../../plugins/security/server';
+import {
+  ElasticsearchServiceSetup,
+  LoggerFactory,
+  SavedObjectsLegacyService,
+} from '../../../../../src/core/server';
+
+// Extend PluginProperties to indicate which plugins are guaranteed to exist
+// due to being marked as dependencies
+interface Plugins extends Hapi.PluginProperties {
+  actions: ActionsPlugin;
+  task_manager: TaskManager;
+  encrypted_saved_objects: EncryptedSavedObjectsPlugin;
+}
+
+export interface Server extends Legacy.Server {
+  plugins: Plugins;
+}
+
+/**
+ * Shim what's we're thinkign setup and start contracts will look like
```

Comment:
typo

Reply:
Fixed in https://github.com/elastic/kibana/pull/46954/commits/920fb6a1d3ad44fc848e3af701342c6cfbe7f44a.